crates/router/src/openapi.rs

```diff
@@ -1,46 +1,60 @@
 #[derive(utoipa::OpenApi)]
 #[openapi(
     info(
-        title = "Juspay Router - API Documentation",
+        title = "Hyperswitch - API Documentation",
         contact(
-            name = "Juspay Support",
-            url = "https://juspay.io",
-            email = "support@juspay.in"
+            name = "Hyperswitch Support",
+            url = "https://hyperswitch.io",
+            email = "hyperswitch@juspay.in"
         ),
         // terms_of_service = "https://www.juspay.io/terms",
         description = r#"
 ## Get started
 
-Juspay Router provides a collection of APIs that enable you to process and manage payments.
+Hyperswitch provides a collection of APIs that enable you to process and manage payments.
 Our APIs accept and return JSON in the HTTP body, and return standard HTTP response codes.
 
 You can consume the APIs directly using your favorite HTTP/REST library.
 
 We have a testing environment referred to "sandbox", which you can setup to test API calls without
-affecting production data.
+affecting production data. Currently, our sandbox environment is live while our production environment is under development and will be available soon. You can sign up on our Dashboard to get API keys to access Hyperswitch API.
 
-### Base URLs
+### Environment
 
 Use the following base URLs when making requests to the APIs:
 
-| Environment   |  Base URL                                            |
-|---------------|------------------------------------------------------|
-| Sandbox       | <https://sandbox.hyperswitch.io>                   |
-| Production    | <https://router.juspay.io>                           |
+| Environment   |  Base URL                          |
+|---------------|------------------------------------|
+| Sandbox       | <https://sandbox.hyperswitch.io>   |
+| Production    | Coming Soon!                       |
 
 ## Authentication
 
-When you sign up on our [dashboard](https://dashboard-hyperswitch.netlify.app) and create a merchant
-account, you are given a secret key (also referred as api-key).
-You may authenticate all API requests with Juspay server by providing the appropriate key in the
+When you sign up on our [dashboard](https://app.hyperswitch.io) and create a merchant
```

Comment:
Please replace the dashboard link with the finalized one elsewhere in the repository.